src/libcpu/src/jit/jit_internal.h

```diff
@@ -108,6 +110,7 @@ class PPCEmuAssembler : public asmjit::X86Assembler
       PPCMemRef(lrMem, lr);
       PPCMemRef(ctrMem, ctr);
 
+      PPCMemRef(ciaMem, cia);
```

Comment:
cia is intentionally not meant to be available through the assembler as it is meant only for debugging purposes and someday should be removed.  Use a.genCia (constant) instead.